build.gradle.kts

```diff
@@ -19,7 +19,6 @@ allprojects {
         google()
         mavenCentral()
         jcenter()
-        maven(url = "https://dl.bintray.com/kotlin/kotlin-dev")
```

Comment:
If you keep this you don't need to add it again when we want to try out RC

Reply:
i feel lik it makes the intent to use stable 1.3.0 clear and readding it is not that much overhead when we have to also change version numbers and ´plugin stuff in settings.gradle.kts
also it makes sure i did not accidentally pull in dependencies from 1.3.0-rc with mismatching coroutine libraries

Reply:
Agreed